prov/verbs/src/verbs_cm.c

```diff
@@ -195,7 +195,7 @@ fi_ibv_msg_ep_accept(struct fid_ep *ep, const void *param, size_t paramlen)
 		return -FI_EINVAL;
 
 	if (!_ep->id->qp) {
-		ret = ep->fid.ops->control(&ep->fid, FI_ENABLE, NULL);
+		ret = fi_control(&ep->fid, FI_ENABLE, NULL);
```

Comment:
Unfortunately no. When updating the new XRC endpoint to use fi_control instead of accessing the function directly I updated the base MSG_EP too. Note though this is not fixing an error, just removing a structure dependency through implementing a better coding practice.

Reply:
Ok, I broke the associated commit into two commit. The existing MSG_EP functions are modified to use fi_control in patch: d68fe1d08e066c916610b8eaa2b13d8f7a4bbd5a